### PR TITLE
assisted installer: add mce and storage e2e tests

### PR DESCRIPTION
**Pull request by rccrdpccl**
this will add deploying storage operator with MCE in the current MCE tests
this will avoid running unnecessary infrastructure for tests, and still tests MCE functionality
in addition, soon storage will be auto-installed when selecting MCE

### CODE REVIEW

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by danmanor** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

File: ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml [hunk 1]
```diff
@@ -307,9 +316,14 @@ tests:
     cluster_profile: packet-assisted
     env:
       ASSISTED_CONFIG: |
-        OLM_OPERATORS=mce
+        OLM_OPERATORS=mce,lvm
         NUM_MASTERS=1
         NUM_WORKERS=0
+        MAKEFILE_TARGET-deploy_nodes_with_install
```
Comment (danmanor) [inline ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:322]: Should be `"` ?

Comment (rccrdpccl) [reply ci-operator/config/openshift/assisted-test-infra/openshift-assisted-test-infra-master.yaml:322]: it should :sweat:

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

File: ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-ref.yaml [hunk 2]
```diff
@@ -17,6 +17,10 @@ ref:
     name: edge-group-brew-registry-redhat-io-pull-secret
     mount_path: /var/run/vault/brew-registry-redhat-io-pull-secret
   dependencies:
+  - name: "test_parallel"
+    env: MAKEFILE_TARGET
+  - name: "setup run"
+    env: MAKEFILE_SETUP_TARGET
```
Comment (adriengentil) [inline ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-ref.yaml:23]: remove this part, the variables are embedded in `ASSISTED_CONFIG`

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by adriengentil** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce
ok issue to get the lease :sweat:

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Comment by rccrdpccl** [comment]
/pj-rehearse pull-ci-openshift-assisted-test-infra-master-e2e-metal-assisted-mce

**Pull request closed without merging**